# HG changeset patch
# User Ben Abbott <bpabbott@mac.com>
# Date 1284933041 14400
#      Sun Sep 19 17:50:41 2010 -0400
# Node ID 728c970a3e7f24cd7ed78d88415d05311564cef1
# Parent  65c14c052fe37d9e706978749e95c693e6bd7343
graphics.h.in: Add displayname property to patch and surface objects.

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,12 @@
+2010-09-19  Ben Abbott <bpabbott@mac.com>
+
+	* graphics.h.in: Add displayname property to patch and surface objects.
+
 2010-09-18  David Bateman  <dbateman@free.fr>
 
 	* graphics.h.in: Remove properties for dead legend code.
 
 2010-09-18  Ben Abbott <bpabbott@mac.com>
 
 	* graphics.cc: Replace pagetype property value custom with <custom>.
 
diff --git a/src/graphics.h.in b/src/graphics.h.in
--- a/src/graphics.h.in
+++ b/src/graphics.h.in
@@ -3573,16 +3573,17 @@ public:
       radio_property erasemode , "{normal}|background|xor|none"
       radio_property linestyle , "{-}|--|:|-.|none"
       double_property linewidth , 0.5
       radio_property marker , "{none}|s|o|x|+|.|*|<|>|v|^|d|p|h|@"
       color_property markeredgecolor , "{auto}|none|flat"
       color_property markerfacecolor , "auto|{none}|flat"
       double_property markersize , 6
       radio_property interpreter , "{tex}|none|latex"
+      string_property displayname , ""
       radio_property alphadatamapping l , "none|{scaled}|direct"
       // hidden properties for limit computation
       row_vector_property xlim hlr , Matrix ()
       row_vector_property ylim hlr , Matrix ()
       row_vector_property zlim hlr , Matrix ()
       row_vector_property clim hlr , Matrix ()
       row_vector_property alim hlr , Matrix ()
       bool_property xliminclude hl , "on"
@@ -3678,16 +3679,17 @@ public:
       color_property edgecolor , color_property (color_values (0, 0, 0), radio_values ("flat|none|interp"))
       radio_property linestyle , "{-}|--|:|-.|none"
       double_property linewidth , 0.5
       radio_property marker , "{none}|s|o|x|+|.|*|<|>|v|^|d|p|h|@"
       color_property markeredgecolor , "{auto}|none"
       color_property markerfacecolor , "auto|{none}"
       double_property markersize , 6
       radio_property interpreter , "{tex}|none|latex"
+      string_property displayname , ""
       array_property alphadata u , Matrix ()
       radio_property alphadatamapping l , "none|direct|{scaled}"
       double_property ambientstrength , 0.3
       radio_property backfacelighting , "unlit|lit|{reverselit}"
       double_property diffusestrength , 0.6
       double_radio_property edgealpha , double_radio_property (1.0, radio_values ("flat|interp"))
       radio_property edgelighting , "{none}|flat|gouraud|phong"
       radio_property erasemode , "{normal}|none|xor|background"
